scripts/generator/graphql_generator/csharp/SDK/Delegates.cs.erb

```diff
@@ -18,6 +18,7 @@ namespace Shopify.Unity.SDK {
     public delegate void MergeFieldDelegate(string field, Dictionary<string, object> into, Dictionary<string, object> responseA, Dictionary<string, object> responseB);
 
     public delegate void ProductsHandler(List<Product> products, ShopifyError error);
```

Comment:
Is there a scenario where we are still using this? I'm thinking we should remove all the non-paginated convenience methods, in which case this could also be removed.

Reply:
So we're using this when we're passing a list of ids to load products.